src/ensembl/src/content/app/browser/track-panel/trackPanelActions.ts

```diff
@@ -144,22 +134,29 @@ export const updatePreviouslyViewedObjectsAndSave = (): ThunkAction<
     ...getActiveGenomePreviouslyViewedObjects(state)
   ];
 
-  const existingIndex = previouslyViewedObjects.findIndex(
-    (previouslyViewedObject) =>
-      previouslyViewedObject.object_id === activeEnsObject.object_id
+  const savedEntitiesWithoutCurrentEntity =
+    previouslyViewedObjects?.filter(
+      (entity) => entity.object_id !== activeEnsObject.object_id
+    ) || [];
+
+  const newObject = {
```

Comment:
Is this `newObject` or `defaultObject`?

Reply:
This is a newly created previously viewed object.